app/jobs/export_vacancies_published_and_updated_since_yesterday_to_dwp_find_a_job_service_job.rb

```diff
@@ -0,0 +1,9 @@
+class ExportVacanciesPublishedAndUpdatedSinceYesterdayToDwpFindAJobServiceJob < ApplicationJob
```

Comment:
I know these class names are very verbose/long but my main goal is that someone can easily see what the job does without having to jump into deeper code.

"The job exports vacancies published and updated since yesterday to DWP Find a Job service"